packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable require-jsdoc */
+import {MediaConnection as MC} from '@webex/internal-media-core';
+
+import LoggerProxy from '../common/logs/logger-proxy';
+
+import {ReceiveSlot, ReceiveSlotId} from './receiveSlot';
+
+export interface ActiveSpeakerPolicyInfo {
+  policy: 'active-speaker';
```

Comment:
yeah we could, but we don't have to as this is our type, we can define it how we want, we don't have to use the underlying WCME types here. Also for some reason my vscode is not dealing very well with MC.Policy.ActiveSpeaker - it doesn't find its value so code completion doesn't work correctly, it may be because MC.Policy is a re-exported type from a dependency 3 layers deep (it's defined in json-multistream lib, then re-exported by WCME and re-exported again by webrtc-media lib) so just using a literal string here works much better for me.
I've just modified the code now to ensure that when we pass the policy value to WCME we use the MC.Policy values